File: app/lib/model/Digit.dart
class Digit {
  String digitId; //代币id
  String chainId; //链id
  String shortName; //缩写名称
  String fullName; //全名
  String balance; //数量
  String money; //金额
  String address; //代币地址
  String contractAddress; //合约地址
  String urlImg; //代币图标icon地址
  bool isVisible; //是否可见
  int decimals; //精度，小数点后的位数
  Digit(chainId) {
    this.chainId = chainId;
  }
}

File: bc/wallets/src/model/wallet/chain/digit.rs
use super::*;

#[repr(C)]
#[derive(Clone)]
pub struct EeeDigit {
    pub status: StatusCode,
    pub digit_id: String,
    pub chain_id: String,
    pub address: Option<String>,
    pub contract_address: Option<String>,
    pub fullname: Option<String>,
    pub shortname: Option<String>,
    pub balance: Option<String>,
    pub is_visible: Option<bool>,
    pub decimal: Option<i64>,
    pub imgurl: Option<String>,
}

#[repr(C)]
#[derive(Clone)]
pub struct EthDigit {
    pub status: StatusCode,
    pub digit_id: String,
    pub chain_id: String,
    pub address: Option<String>,
    pub contract_address: Option<String>,
    pub fullname: Option<String>,
    pub shortname: Option<String>,
    pub balance: Option<String>,
    pub is_visible: Option<bool>,
    pub decimal: Option<i64>,
    pub imgurl: Option<String>,
}

#[repr(C)]
#[derive(Clone)]
pub struct BtcDigit {
    pub status: StatusCode,
    pub digit_id: String,
    pub chain_id: String,
    pub address: Option<String>,
    pub contract_address: Option<String>,
    pub fullname: Option<String>,
    pub shortname: Option<String>,
    pub balance: Option<String>,
    pub is_visible: Option<bool>,
    pub decimal: Option<i64>,
    pub imgurl: Option<String>,
}
